python/ray/tune/trainable/trainable.py

```diff
@@ -1146,23 +1147,34 @@ def load_checkpoint(self, checkpoint: Union[Dict, str]):
         The directory structure under the checkpoint_dir provided to
         ``Trainable.save_checkpoint`` is preserved.
 
-        See the example below.
+        See the examples below.
 
         Example:
             >>> from ray.tune.trainable import Trainable
             >>> class Example(Trainable):
             ...    def save_checkpoint(self, checkpoint_path):
-            ...        print(checkpoint_path)
-            ...        return os.path.join(checkpoint_path, "my/check/point")
-            ...    def load_checkpoint(self, checkpoint):
-            ...        print(checkpoint)
+            ...        my_checkpoint_path = os.path.join(checkpoint_path, "my/path")
+            ...        return my_checkpoint_path
+            ...    def load_checkpoint(self, my_checkpoint_path):
+            ...        print(my_checkpoint_path)
             >>> trainer = Example()
             >>> # This is used when PAUSED.
             >>> obj = trainer.save_to_object() # doctest: +SKIP
-            <logdir>/tmpc8k_c_6hsave_to_object/checkpoint_0/my/check/point
+            <logdir>/tmpc8k_c_6hsave_to_object/checkpoint_0/my/path
```

Comment:
I wonder if we should be consistent and use something like `<checkpoint_path>` here, since we're already using `<logdir>`? 

The same goes for `/tmp/mypath`, resp. `~/ray_results/` in tune-checkpoints.rst above. should this maybe also be `<logdir>`? Not sure about this one, I feel like we need a good naming convention for the base log dir.

Reply:
I think it makes sense to use `<logdir>` because we didn't specify it in this example. I set the `my/path` checkpoint path explicitly in the `save_checkpoint`. For the `/tmp/mypath` example, I also set the `local_dir="/tmp/mypath"` in the config, so it made sense to use it directly.